contracts/DualGovernance.sol

```diff
@@ -143,19 +151,51 @@ contract DualGovernance is IGovernance, ConfigurationProvider {
     // Tiebreaker Protection
     // ---
 
-    function tiebreakerSchedule(uint256 proposalId) external {
+    function tiebreakerApproveProposal(uint256 proposalId) external {
+        _checkTiebreakerCommittee(msg.sender);
+        _dgState.checkTiebreak(CONFIG);
+        if (_tiebreakerProposalApprovalTimestamp[proposalId] > 0) {
+            revert ProposalAlreadyApproved(proposalId);
+        }
+
+        _tiebreakerProposalApprovalTimestamp[proposalId] = block.timestamp;
+        emit ProposalApprovedForExecition(proposalId);
+    }
+
+    function tiebreakerApproveSealableResume(address sealable) external {
         _checkTiebreakerCommittee(msg.sender);
         _dgState.checkTiebreak(CONFIG);
+        Proposer memory proposer = _proposers.get(msg.sender);
+        ExecutorCall[] memory calls = new ExecutorCall[](1);
+        calls[0] = ExecutorCall(sealable, 0, abi.encodeWithSignature("resume()"));
+        uint256 proposalId = TIMELOCK.submit(proposer.executor, calls);
+        _tiebreakerProposalApprovalTimestamp[proposalId] = block.timestamp;
+        emit ProposalApprovedForExecition(proposalId);
+        emit SealableResumeApproved(sealable);
+    }
+
+    function tiebreakerSchedule(uint256 proposalId) external {
+        _dgState.checkTiebreak(CONFIG);
+        if (_tiebreakerProposalApprovalTimestamp[proposalId] == 0) {
+            revert ProposalIsNotApprovedForExecution(proposalId);
+        }
+        if (_tiebreakerProposalApprovalTimestamp[proposalId] + _tiebreakerProposalApprovalTimelock > block.timestamp) {
+            revert TiebreakerTimelockIsNotPassed(proposalId);
+        }
         TIMELOCK.schedule(proposalId);
     }
 
     function setTiebreakerProtection(address newTiebreaker) external {
         _checkAdminExecutor(msg.sender);
-        address oldTiebreaker = _tiebreaker;
-        if (newTiebreaker != oldTiebreaker) {
-            _tiebreaker = newTiebreaker;
-            emit TiebreakerSet(newTiebreaker);
+        if (_tiebreaker == newTiebreaker) {
+            revert TieBreakerAddressIsSame();
+        }
+        if (_tiebreaker != address(0)) {
+            _proposers.unregister(CONFIG, _tiebreaker);
         }
+        _tiebreaker = newTiebreaker;
+        _proposers.register(newTiebreaker, CONFIG.ADMIN_EXECUTOR()); // TODO: check what executor should be. Reseal executor?
```

Comment:
Because we need to propose "resume" proposal and execute it. I agree that it's not the correct behavior so we need to specify the proposer for this case.